src/main/cpp/t06_longest.cpp

```diff
@@ -22,5 +22,22 @@
 using namespace std;
 
 int t06_longest() {
-
+     string s, w, max;
+    getline(cin, s);
+   int a, b, c;
+   a = s.size();
+   b =w.size();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'b' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4675615)